packages/grid/_modules_/grid/components/toolbar/GridFilterToolbarButton.tsx

```diff
@@ -1,6 +1,8 @@
+import { makeStyles } from '@material-ui/core';
```

Comment:
I think that we have already seen this, no top-level import

```suggestion
import { makeStyles } from '@material-ui/core/styles';
```

Reply:
Updated